aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -106,7 +106,7 @@ Tensor empty_cpu(IntArrayRef size, const TensorOptions& options, c10::optional<c
     allocator,
     /*resizeable=*/true);
 
-  auto tensor = detail::make_tensor<TensorImpl>(storage_impl, at::CPUTensorId());
+  auto tensor = detail::make_tensor<TensorImpl>(std::move(storage_impl), at::CPUTensorId());
```

Comment:
Does this improve perf in any noticeable way? Is the compiler smart enough to do the move on its own?

Reply:
This change doesn't change perf much on its own because the overhead is a death-by-1000-cuts situation, but I think it's still useful